burn-wgpu/src/ops/numeric.rs

```diff
@@ -170,20 +168,33 @@ impl<G: GraphicsApi> NumericOps<G> {
         tensor: WgpuTensor<E, D>,
         dim: usize,
     ) -> WgpuTensor<E, D> {
-        reduction_sum_dim(tensor, dim)
+        reduction_dim::<SumDim, E, D>(tensor, dim)
     }
 
     pub fn mean<E: WgpuElement + Element, const D: usize>(
         tensor: WgpuTensor<E, D>,
     ) -> WgpuTensor<E, 1> {
         let num_elems = tensor.shape.num_elements();
-        Self::div_scalar(reduction_sum(tensor), (num_elems as f32).elem())
+        Self::div_scalar(Self::sum(tensor), (num_elems as f32).elem())
     }
 
     pub fn mean_dim<E: WgpuElement + Element, const D: usize>(
         tensor: WgpuTensor<E, D>,
         dim: usize,
     ) -> WgpuTensor<E, D> {
-        reduction_mean_dim(tensor, dim)
+        reduction_dim::<MeanDim, E, D>(tensor, dim)
+    }
+
+    pub fn argmax<E: WgpuElement + Element, I: WgpuElement, const D: usize>(
+        tensor: WgpuTensor<E, D>,
+        dim: usize,
+    ) -> WgpuTensor<I, D> {
+        reduction_args_dim::<ArgsMax, E, I, D>(tensor, dim)
+    }
+    pub fn argmin<E: WgpuElement + Element, I: WgpuElement, const D: usize>(
```

Comment:
No space between those functions lol. Isn't it auto-formatted?

Reply:
Yeah I think he missed it. Also currently WGPU is not hooked up to the CI.

Reply:
Nop, it's not a clippy not a fmt error/warning, eventhough I like to add spacing between functions.